sessionctx/context.go

```diff
@@ -28,6 +28,14 @@ import (
 	"github.com/pingcap/tipb/go-binlog"
 )
 
+// InfoschemaMetaVersion is a workaround. Due to circular dependency,
+// can not return the complete interface. But SchemaMetaVersion is widely used for logging.
+// So we give a convenience for that.
+// FIXME: remove this interface
+type InfoschemaMetaVersion interface {
+	SchemaMetaVersion() int64
```

Comment:
For interface used as a tool to avoid circular, we can put it in the util package.
But for this, it's ok.